daemon/entrypoint.sh

```diff
@@ -65,6 +65,13 @@ chown ceph. /var/run/ceph
 # based on the data store
 case "$KV_TYPE" in
    etcd|consul)
+      # if populate.sh script found in /etc/ceph/ use that
+      if [[ -e "/etc/ceph/populate.sh" ]]; then
```

Comment:
What about having a `KV_POPULATE=(true|false)` option in order to enable/disable this feature.
I already populate my KV store outside the container and I don't want to add extra pressure to `etcd` by scanning all the keys again.